ValveResourceFormat/IO/ModelExtract.cs

```diff
@@ -303,11 +303,36 @@ Lazy<KVObject> MakeLazyList(string className)
 
             foreach (var (physHull, fileName) in PhysHullsToExtract)
             {
+                var attributes = physAggregateData.CollisionAttributes[physHull.CollisionAttributeIndex];
+                var tags = attributes.GetArray<string>("m_InteractAsStrings") ?? attributes.GetArray<string>("m_PhysicsTagStrings");
+                var tooltexture = MapExtract.GetToolTextureShortenedName_ForInteractStrings(new HashSet<string>(tags));
+
+                // dont extract tool brushes for map decompile phys
+                if (Type == ModelExtractType.Map_PhysicsToRenderMesh)
+                {
+                    if (tooltexture != "nodraw")
+                    {
+                        continue;
+                    }
+                }
+
                 HandlePhysMeshNode(physHull, fileName);
             }
 
             foreach (var (physMesh, fileName) in PhysMeshesToExtract)
             {
+                var attributes = physAggregateData.CollisionAttributes[physMesh.CollisionAttributeIndex];
+                var tags = attributes.GetArray<string>("m_InteractAsStrings") ?? attributes.GetArray<string>("m_PhysicsTagStrings");
+                var tooltexture = MapExtract.GetToolTextureShortenedName_ForInteractStrings(new HashSet<string>(tags));
+
+                // dont extract tool brushes for map decompile phys
+                if (Type == ModelExtractType.Map_PhysicsToRenderMesh)
+                {
+                    if (tooltexture != "nodraw")
+                    {
+                        continue;
+                    }
```

Comment:
Don't need these checks if model isn't added to ModelsToExtract. 
The lines that do
`//convert entity to hammer mesh` 
should handle that.